go/host/client_api_eth.go

```diff
@@ -106,12 +115,11 @@ func (api *EthereumAPI) SendRawTransaction(_ context.Context, encryptedParams co
 func (api *EthereumAPI) GetCode(_ context.Context, address gethcommon.Address, blockNrOrHash rpc.BlockNumberOrHash) (hexutil.Bytes, error) {
 	rollupHeight, ok := blockNrOrHash.Number()
 	if ok {
-		rollup, err := api.host.EnclaveClient.GetRollupByHeight(rollupHeight.Int64())
```

Comment:
Tangential to the changes here, but saw a good opportunity to avoid a hop to the enclave.